config/default/.htaccess

```diff
@@ -0,0 +1,27 @@
+# Deny all requests from Apache 2.4+.
```

Comment:
We don't use Apache, so this directory could be deleted?

Reply:
Oh this is probably just a stray file, I'm not even sure why it's included.

I'll remove it, thank you for spotting it.

Reply:
Removed, ty :)